core/lomas_core/models/requests.py

```diff
@@ -1,29 +1,34 @@
 from typing import List, Optional, Union
 
+from pydantic import BaseModel, ConfigDict, Field
+
 from lomas_core.constants import (
     DPLibraries,
     SSynthGanSynthesizer,
     SSynthMarginalSynthesizer,
 )
 from lomas_core.error_handler import InternalServerException
-from pydantic import BaseModel, Field
 
 
-class GetDbData(BaseModel):
+class LomasRequestModel(BaseModel):
```

Comment:
could dataset_name already be in there ?
then RequestModel can be removed (and maybe also GetDsData)

class QueryModel(LomasRequestModel):
   ...

Reply:
you are right, i made the change in the last commit! Thank you